packages/accordion/src/vaadin-accordion-panel.js

```diff
@@ -142,6 +147,50 @@ class AccordionPanel extends DetailsMixin(
   _setAriaDisabled() {
     // The `aria-disabled` is set on the details summary.
   }
+
+  /** @private */
+  _initSummary() {
+    this._summaryController = new SummaryController(this);
+    this.addController(this._summaryController);
+
+    // Wait for heading element render to complete
+    afterNextRender(this, () => {
+      this._toggleElement = this.focusElement.$.button;
+    });
+  }
+
+  /** @private */
+  _initContent() {
+    this._contentController = new ContentController(this);
+    this._contentController.addEventListener('slot-content-changed', (event) => {
+      // Store nodes to toggle `aria-hidden` attribute
+      const content = event.target.nodes || [];
+      this._contentElements = content;
+
+      // See https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+      const node = content[0];
+      if (node) {
+        node.setAttribute('role', 'region');
+        node.setAttribute('aria-labelledby', this.focusElement.id);
+      }
+
+      if (node && node.id) {
+        this.focusElement.setAttribute('aria-controls', node.id);
+      } else {
+        this.focusElement.removeAttribute('aria-controls');
```

Comment:
I expected removing the content element to result in the removal of the "aria-controls" attribute. It seems that `node` is present in that case.

https://user-images.githubusercontent.com/1222264/206671806-bc37028b-6ccf-4c99-97ce-df7abfd681d5.mp4

Reply:
Thanks, fixed.